MANIFEST.in

```diff
@@ -39,6 +40,14 @@ global-exclude .DS_Store
 global-exclude .git*
 global-exclude \#*
 
+global-exclude *.c
+global-exclude *.cpp
+global-exclude *.h
```

Comment:
We need to keep the  .c and .h for vendored code e.g. parsers etc

Reply:
`graft pandas/_libs/src` seems to work ok (and need to do the same for np_datetime.c and another couple of files)